CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
+
+## Build and test the documentation
+
+```bash
+cd docs
+make install
```

Comment:
If we run `make install; make run`  _build folder does not exists and following error appears:
```
make run
. .sphinx/venv/bin/activate; sphinx-autobuild -c . -b dirhtml "." "_build"
Traceback (most recent call last):
  File "/home/gatici/charmed5g/charmed-oai-ran/docs/.sphinx/venv/bin/sphinx-autobuild", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/gatici/charmed5g/charmed-oai-ran/docs/.sphinx/venv/lib/python3.11/site-packages/sphinx_autobuild/__main__.py", line 34, in main
    args, build_args = _parse_args(list(argv))
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gatici/charmed5g/charmed-oai-ran/docs/.sphinx/venv/lib/python3.11/site-packages/sphinx_autobuild/__main__.py", line 101, in _parse_args
    args.outdir = Path(sphinx_args.outputdir).resolve(strict=True)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/pathlib.py", line 994, in resolve
    s = os.path.realpath(self, strict=strict)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 416, in realpath
  File "<frozen posixpath>", line 451, in _joinrealpath
FileNotFoundError: [Errno 2] No such file or directory: '_build'
make: *** [Makefile:33: run] Error 1
```
Instead if we run `make install; make html; make run` the page is served.

Reply:
Should be fixed now